packages/cli/src/lib/bundler/transforms.ts

```diff
@@ -32,6 +32,32 @@ export const transforms = (options: TransformOptions): Transforms => {
 
   const extraTransforms = isDev ? ['react-hot-loader'] : [];
 
+  // Make sure that imported css styles are included before style
+  // elements created by makeStyle to keep a stable insertion
+  // order which results in a stable evaluation order of the css
+  // selectors.
+  //
+  // insert is executed in the browser without transformation,
+  // avoid using new JS features here!
+  function insertElementAfterTitle(element: any) {
+    const head = document.head;
+    // An alternative is to include an explicit insertion point
+    // like <noscript id="css-insertion-point"></noscript> in the
+    // index.html, and retrieve it here like:
+    // var insertionPoint = head.getElementById(
+    //   'css-insertion-point',
+    // );
+    // But that would be a breaking change, therefore we start
+    // with abusing the <title> tag.
```

Comment:
Could we prepend on `<head>` instead?

Reply:
Even better! Users could miss the title. If we add the static css at the beginning and the dynamic at the end, we should be fine.

Reply:
There is just a small problem, but I think that should have any effect. There is a style tag that styles `#root` (`min-height: 100%`) that is now included later then before.

Reply:
On, no, wait... Not sure if we want to do this. This would change the order of the elements:

With `<title>`:
```
<head>
  …
  <staticstyle1>
  <staticstyle2>
  <staticstyle3>
  <title> 
  …
  <dynamicstyle1>
  <dynamicstyle2>
<head>
```

With `<head>`:
```
<head>
  <staticstyle3>
  <staticstyle2>
  <staticstyle1>
  …
  <title> 
  …
  <dynamicstyle1>
  <dynamicstyle2>
<head>
```

In that case the order of the static styles is mabe not the way you would expect it. Even though is quite undefined behavior anway. Most important is the order between static and dynamic styles.